KOGITO-7263: [KSW-Guides] Orchestrating third party services with OAuth2 authentication

wmedvede opened the pull request:
- Adds the corresponding card to the index.adoc

<!-- Please don't forget your JIRA link -->
**JIRA:**

<!-- If you don't have a JIRA link, please provide a short description of what this PR does -->
**Description:**

<!-- Link to related PRs: -->

Please make sure that your PR meets the following requirements:

- [X] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [X] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [X] The nav.adoc file has a link to this guide in the proper category

hmanwani-rh commented on serverlessworkflow/modules/ROOT/pages/index.adoc:
```diff
@@ -71,6 +71,14 @@ xref:security/authention-support-for-openapi-services.adoc[Authentication for Op
+[.card-title]
+xref:security/orchestrating-third-party-services-with-oauth2.adoc[Orchestrating third party services with OAuth2 authentication]
+[.card-description]
+Orchestrating third party services with OAuth2 authentication
```
```suggestion
Orchestration of third-party services using OAuth 2.0 authentication
```

wmedvede replied:
@hmanwani-rh yes, looks good. 
I didn't add your suggested change directly because the change must be applied in other files too, but your suggestion was added on my additional commit.
Thanks.

hmanwani-rh reviewed (commented):
@wmedvede Added a suggestion below. Please let me know if you've any question.

wmedvede commented:
@hmanwani-rh I've removed the "card" I've added as suggested by @ricardozanini , and I've kept the naming change to Orchestration instead Orchestrating.
So this PR, while not blocker, still makes sense.
